mountpoint-s3/src/fs.rs

```diff
@@ -406,6 +406,10 @@ where
             .superblock
             .create(&self.client, parent, name, InodeKind::File)
             .await?;
+        // Log newly created Inode number
+        let inode_nr = lookup.inode.ino();
+        debug!("mknod: created inode number {inode_nr}");
```

Comment:
I'd take some inspiration from this line:

https://github.com/awslabs/mountpoint-s3/blob/534918e96337dee222b158df9d4bd92a05d791b9/mountpoint-s3/src/fs.rs#L342

I'd probably write something like this, such that it matches the format of other logs. I'd also drop the comment since its implied.

```suggestion
        debug!(ino = lookup.inode.ino(), "new inode created");
```

This should format the field nicely, and `tracing` library should ensure context of `mknod` is already captured.

Hopefully it'll look something like this:

```
2024-11-07T15:02:24.409045Z DEBUG mknod{req=12 parent=1 name="hello.md"}: mountpoint_s3_client::fs: new inode created ino=5
```